src/classes/QueryBuilder.cls

```diff
@@ -2,41 +2,63 @@
 * This file is part of the Nebula Framework project, released under the MIT License.             *
 * See LICENSE file or go to https://github.com/jongpie/NebulaFramework for full license details. *
 *************************************************************************************************/
-public without sharing class QueryBuilder extends NebulaCore implements IQueryBuilder {
+public class QueryBuilder extends NebulaCore implements IQueryBuilder {
```

Comment:
This class seems to have grown very large and is becoming difficult to maintain. Please consider refactoring to reduce the number of methods.